src/common/snippets/include/snippets/common.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ie/ie_common.h>
+#include <openvino/util/log.hpp>
+
+/**
+ * @def SNIPPETS_THROW
+ * @brief A macro used for snippet transformations
+ * to throw specified exception with a description.
+ */
+#define SNIPPETS_THROW(...) IE_THROW(__VA_ARGS__) << "[SNIPPETS] "
```

Comment:
I'm not sure it's a good idea to use IE exceptions here, because previously we had some issues caused by passing exceptions between different libraries (and casting the exception to a proper type). These are rare cases in specific environments, but probably better to avoid anyway. 

As a minimal solution, we can restrict to using IE_THROW without arguments, since it's almost always called like this anyway.

Reply:
Thank you, to be agreed with @dmitry-gorokhov